olive/engine/engine.py

```diff
@@ -65,15 +65,15 @@ def __init__(
         elif self._config.host is not None:
             self.host = self._config.host.create_system()
         else:
-            self.host = LocalSystem()
+            self.host = LocalSystem([Device.CPU])
 
         # engine target
         if target is not None:
             self.target = target
         elif self._config.target is not None:
             self.target = self._config.target.create_system()
         else:
-            self.target = LocalSystem()
+            self.target = LocalSystem([Device.CPU])
```

Comment:
I suppose this is needless since the default CPU is already handled.

Reply:
I wanted the default to be clear since the default value for the LocalSystem accelerator is None and it generates this warning https://github.com/microsoft/Olive/blob/main/olive/engine/engine.py#L96.

Reply:
make sense then